pkg/controllers/v1alpha2/mpi_job_controller.go

```diff
@@ -1198,6 +1201,16 @@ func (c *MPIJobController) newLauncher(mpiJob *kubeflow.MPIJob, kubectlDeliveryI
 				MountPath: configMountPath,
 			},
 		},
+		Resources: corev1.ResourceRequirements{
```

Comment:
Should we expose this to `MPIJobSpec` similar to `MPIReplicaSpecs`? What if user needs resources other than cpu and memory?

Reply:
This is only for init container though, not sure if user needs to customize it.

Reply:
What if their requirements are different, e.g. larger than these default values?

Reply:
In what cases would user want to customize the init container?

Reply:
@gaocegege @rongou Any thoughts on this?

Reply:
Seems fine. The init container really just copies the `kubectl` binary and does some initial checking. Resource usage should be minimal.